src/core/ExtensionsManager.ts

```diff
@@ -70,6 +80,11 @@ export class ExtensionsManager {
             this.#pmTransformers = options.pmTransformers;
         }
 
+        if (options.allowDynamicModifiers) {
```

Comment:
Why is needed `allowDynamicModifiers` option?
U can use `dynamicModifiers` option only

Reply:
fixed